packages/aura-language-server/src/tern-server/__tests__/__snapshots__/ternCompletion.spec.ts.snap

```diff
@@ -5,8 +5,30 @@ exports[`tern completions 1`] = `
   "isIncomplete": true,
   "items": [
     {
-      "detail": "AuraInstance",
-      "documentation": "#include aura.Aura_export",
+      "detail": "{enqueueAction, error, hasErrors, lastKnownError, test}",
```

Comment:
as far as I can tell, the unit test was not actually loading all of the language files from resources, probably due to its path issues.  My "find the path" search seemed to return a lot more in the completion details

I could be completely wrong about this, please review